content/en/service_management/events/guides/email.md

```diff
@@ -21,10 +21,19 @@ Before you can send events with email, you need a dedicated email address from D
 2. From the **Account** menu at the bottom left, select **Organization Settings**.
 3. Click the **Events API emails** tab.
 4. Choose the format for your messages from the **Format** dropdown (`Plain text` or `JSON`).
-5. Click the **Create Email** button.
+5. [Optional] Define other attributes (more information in the next section).
+6. Click the **Create Email** button.
 
 The **Events API emails** section displays all the emails available for your applications and who created them.
 
+### Attributes definition
+| Name | Description | Example |
+|---|---|---|
+| Description | A short text for clarifying the email usage | "Used for MyService notifications" |
+| Tags | List of tags that to be appended to each event received through the email. If other tags are present in the JSON message, they are all added | `tag1:val1`, `tag2:val2` |
+| Recipients | List of handles to be added in the beginning of the texts for all events created through the email, without `@` prefix. See more: [Notifications](/service_management/events/explorer/notifications). | `my@email.com`, `@slack-acc-ch` |
```

Comment:
```suggestion
| Recipients | List of handles to be added to the beginning of the message for all events created through the email, without `@` prefix. For more information, see [Notification recipients][7]. | `my@email.com`, `slack-acc-ch` |
```

Reply:
Suggesting a few updates here:
- Changed the link format to be a reference link per the [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#links). Additionally, it seems like `/monitors/notify/#notification-recipients` might be a more informative reference in this case (the `/service_management/events/explorer/notifications` page also includes a link to this page). What do you think?
- The instructions mention "without `@` prefix", but one of the examples includes the `@` prefix. I've removed it in this suggestion, but let me know if I'm misunderstanding something here.